tool/tsh/tsh.go

```diff
@@ -225,6 +225,10 @@ func Run(args []string, underTest bool) {
 	show := app.Command("show", "Read an identity from file and print to stdout").Hidden()
 	show.Arg("identity_file", "The file containing a public key or a certificate").Required().StringVar(&cf.IdentityFileIn)
 
+	// The status command shows which proxy the user is logged into and metadata
+	// about the certificate.
+	status := app.Command("status", "Show proxies user is connected to.")
```

Comment:
remove `.` as we don't have it elsewhere, also I would change the wording as status is a bit more generic command, ask @kontsevoy may be about wording